packages/mentions-wrapper/src/__tests__/UserList.spec.js

```diff
@@ -56,15 +57,25 @@ describe('Mentions: MentionsWrapper > UserList component', () => {
                 .find(ListItemText)
                 .first()
                 .props().primary
-        ).toEqual('Johnny Cash (sue)');
+        ).toEqual('Eric Clapton (slowhand)');
+    });
+
+    it('should sort the mentions in alphabetical order by firstname', () => {
+        const sortedUsers = sortBy(props.users, [userName => userName.displayName]);
```

Comment:
I don't think we should use the sortBy function in the test since that is the exact code in the implementation, thus not really testing anything. An alternative would be to spy on sortBy and ensure that it is called with the correct arguments. This is also a case where you could hardcode the expected result.

One other thing I want to mention regarding this test file is in setting the properties lines 24-37. I would define the users array separately, then in the props you will have:
```
{
users,
selectedUsers: users[2],
}
```